wpiunits/src/generate/main/java/Measure-interface.java.jinja

```diff
@@ -43,6 +43,12 @@ public interface {{ helpers['type_decl'](name) }} extends Measure<{{ helpers['mt
     return ({{ helpers['type_usage'](name) }}) unit().ofBaseUnits(0 - baseUnitMagnitude());
   }
 
+  @Override
+  @Deprecated(since = "2025", forRemoval = true)
+  default {{ helpers['type_usage'](name) }} negate() {
```

Comment:
Adding `"deprecation"` and `"removal"` to the class-level `@SuppressWarnings` annotation could be helpful